### PR TITLE
Uses clearer language in the README

### PR DESCRIPTION
**Pull request by mikesol**
- Fixes passthrough, as it is a contresens as currently used
- Change the note on required query strings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,9 +250,9 @@ unmock. nock('http://myapp.iriscouch.com')
   })
 ```
 
-### Passthrough
+### Trivial API calls
 
-For "boring" API calls where you are just passing through information, you can instruct unmock to serve random `200` resposnes to those requests using `tldr`.
+For "trivial" API calls where you are just passing through information, you can instruct unmock to serve random `200` resposnes to those requests using `tldr`.
```
Comment (carolstran) [inline README.md]: A few notes about this section...

1) Can we remove `just`? I think the sentence works without it and this language is often considered condescending in documentation. We could also reword it to something like `For "trivial" API calls where the goal is to pass through information...` 🤔 

2) Typo --> `responses`

File: README.md [hunk 2]
```diff
@@ -250,9 +250,9 @@ unmock. nock('http://myapp.iriscouch.com')
   })
 ```
 
-### Passthrough
+### Trivial API calls
```
Comment (carolstran) [inline README.md]: I also wonder if there's a better header for this... we can keep for the sake of this PR, but might revisit later.

Comment (mikesol) [reply README.md]: I just changed it to "Ignorable", which is perhaps not as easy for a non-native speaker as it is a neologism, but it may get the idea across better.

Comment (carolstran) [reply README.md]: Yeah maybe... let's sit on it for a while and if something else comes to mind we can always change it.

Merging now 🚀
